backend/packages/worker/src/worker/parsing/parser.py

```
@@ -0,0 +1,50 @@
+from typing import Protocol, List
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)